### PR TITLE
Set VS toolset binaries to 32bitpref to workaround VS build bug

### PR DESCRIPTION
**Pull request by agocke**

### CODE REVIEW

File: src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb [hunk 1]
```diff
@@ -1042,6 +1042,12 @@ Public Class BuildDevDivInsertionFiles
         ' No duplicates are allowed
         filesToInsert.GroupBy(Function(x) x).All(Function(g) g.Count() = 1)
 
+
```
Comment (jaredpar) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: Double blank line.

File: src/Setup/DevDivPackages/Roslyn.proj [hunk 2]
```diff
@@ -12,7 +12,7 @@
       <!-- TFS build number isn't set on CI server -->
       <Version Condition="'$(NuGetPerBuildPreReleaseVersion)' == ''">$(NuGetReleaseVersion)-cibuild</Version>
     </NuSpec>
-    <NuSpec Include="$(InsertionFilesDir)\VS.Tools.Roslyn.nuspec">
+    <NuSpec Include="$(InsertionFilesDir)\VS_Toolset_files\VS.Tools.Roslyn.nuspec">
```
Comment (jaredpar) [inline src/Setup/DevDivPackages/Roslyn.proj]: Nit: capitalize f in `_files` for consistency with named constant in the VB code.

File: src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb [hunk 3]
```diff
@@ -1066,14 +1097,14 @@ set DEVPATH=%RoslynToolsRoot%;%DEVPATH%"
                           <version>0.0</version>
                       </metadata>
                       <files>
-                          <file src="Init.cmd"/>
+                          <file src=<%= Path.Combine(outputFolder, " Init.cmd") %>/>
```
Comment (jaredpar) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: Errant space after the quote is breaking the microbuild leg

**Comment by agocke** [comment]
test windows_release_unit32_prtest please

**Comment by agocke** [comment]
@jaredpar Should be fixed now.

File: src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb [hunk 4]
```diff
@@ -1053,9 +1050,34 @@ set RoslynToolsRoot=%~dp0
...
 
         File.WriteAllText(
-            Path.Combine(_binDirectory, "Init.cmd"),
+            Path.Combine(outputFolder, "Init.cmd"),
             fileContents)
 
+        ' Copy all dependent compiler files to the output directory
+        ' It is most important to have isolated copies of the compiler
+        ' exes (csc, vbc, vbcscompiler) since we are going to mark them
+        ' 32-bit only to work around problems with the VS build.
```
Comment (jaredpar) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: Here is the bug tracknig the underlying problem

https://devdiv.visualstudio.com/web/wi.aspx?pcguid=011b8bdf-6d56-4f87-be0d-0092136884d9&id=277324

File: src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb [hunk 5]
```diff
@@ -1053,9 +1050,34 @@ set RoslynToolsRoot=%~dp0
...
+        ' These binaries should never ship anywhere other than the VS toolset
+        Dim corFlagsPath = Path.Combine(
+            Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86),
+            "Microsoft SDKs/Windows/v10.0A/bin/NETFX 4.6 Tools/CorFlags.exe")
```
Comment (jaredpar) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: The key for @jaredpar is to make sure my sadness floor stays level.  This way I can't get sadder.

Comment (agocke) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: I don't have a better way of locating this DLL. :/ Suggestions?

File: src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb (hunk 4 above)
Comment (agocke) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: Added a link to the bug (#17864).

File: src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb (hunk 5 above)
Comment (jaredpar) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: There isn't a NuGet I'm aware of or a direct download.  Hopefully this is just a very temporary item.  I'd say leave it this for now and if we hit a prob here we'll make a NuGet.

**Comment by agocke** [comment]
test windows_release_unit32_prtest please

**Comment by agocke** [comment]
@jasonmalinowski Any more comments for this PR?

**Comment by jaredpar** [comment]
Chatted with @jasonmalinowski offline, he doesn't have any blocking feedback.

File: src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb (hunk 5 above)
Comment (jaredpar) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: And hit a problem.  This isn't present on clean windows machines hence my xcopy change ran face first into it.  😦

**Pull request merged**